examples/local/configmap.tmpl.yaml

```diff
@@ -4,12 +4,13 @@ metadata:
   name: aws-operator-configmap
 data:
   config.yml: |
```

Comment:
It is now called `.yaml`. See https://github.com/giantswarm/aws-operator/blob/c834fa6df95e720606b1a9ca1a9c0a21ffd18e27/helm/aws-operator-chart/templates/configmap.yaml#L7.